src/metabase/driver/redshift.clj

```diff
@@ -90,6 +90,7 @@
                                                      :name         "db"
                                                      :placeholder  (tru "toucan_sightings"))
                                                    driver/default-user-details
+                                                   driver/default-password-details
                                                    driver/default-port-details]))
```

Comment:
Where I messed up is adding `driver/default-port-details` rather than `driver/default-password-details`. I'll push up another commit that removes the second port entry.